mailparser/mailparser.py

```diff
@@ -226,8 +226,12 @@ def message_id(self):
 
     @property
     def to_(self):
-        return decode_header_part(
-            self._message.get('to', self._message.get('delivered-to')))
+        to_ = self._message.get('to')        
+        if not to_:
+            return None
+        else:
```

Comment:
Ok I understood. I can see the *X-* headers, maybe they are used in Java server. These header are not valid, you can see in RFC http://stackoverflow.com/questions/14469110/what-do-x-headers-in-mails-stand-for

You don't see to because these mails have X-to. Your snippet doesn't resolve your problem.

Can you tell me the source of those mails?